src/ore/src/tracing.rs

```diff
@@ -316,12 +317,31 @@ where
             .install_batch(opentelemetry::runtime::Tokio)
             .unwrap();
         let (filter, filter_handle) = reload::Layer::new(if otel_config.start_enabled {
-            otel_config.filter
+            let targets = Targets::default()
+                // By default we turn off tracing from the following crates, because they
+                // have long-lived Spans, which OpenTelemetry does not handle well. We
+                // specifically apply the otel_config _after_ these defaults, to allow the
+                // otel_config to override this setting, if it's desired.
+                //
+                // Note: users should feel free to add more crates here if we find more
+                // with long lived Spans.
+                .with_targets([("h2", LevelFilter::OFF), ("hyper", LevelFilter::OFF)]);
+
+            // Apply the user provided config
+            match otel_config.filter.default_level() {
+                Some(defaul_level) => targets.with_default(defaul_level),
+                _ => targets,
+            }
+            .with_targets(otel_config.filter)
```

Comment:
can we just to `otel_config.filter.with_targets(h2 and hyper stuff)`?

Reply:
I didn't at first because I thought the `h2 and hyper stuff` would override if `h2` or `hyper` was set in `otel_config`, but through testing apparently it doesn't! So I'll make this update

Reply:
Ah I see :(